package.json

```diff
@@ -13,7 +13,8 @@
     "dev": "npm run watch & npm run onchange",
     "prepublish": "npm ls && npm test",
     "example:compile": "gulp",
-    "example:watch": "gulp watch:example"
+    "example:watch": "gulp watch:example",
+    "build_single": "node single_build_globals.js"
```

Comment:
Sure. The script should probably also live somewhere else and always build the same file. I didn't know how to organise it to be honest, just wanted to share the build script.